src/askContainer/commands/viewAllSkills.ts

```diff
@@ -146,6 +150,49 @@ export class ViewAllSkillsCommand extends AbstractCommand<void> {
         }
     }
 
+    private async checkAndSetQpItems(
+        context: vscode.ExtensionContext,
+        qp: vscode.QuickPick<vscode.QuickPickItem>
+    ): Promise<void> {
+        Logger.verbose(`Calling method: ${this.commandName}.calculateTimeExceedHourly, args:`, qp);
```

Comment:
nit: wrong method name in the logger statement

Reply:
Updated it.